### PR TITLE
feat(excludedMetrics): add a configurable excludedMetrics list 

### PR DESCRIPTION
**Pull request by shlomodaari**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -220,6 +222,7 @@ spinnaker:
                   # By default this plugin adds a set of sane default tags to help with observability best practices, you can disable those here
                   # Optional, Default: false
                   defaultTagsDisabled: false
+                  excludedMetrics: ["Metric1","Metric2"] #configure Metrics to exclude
```
Comment (jasonmcintosh) [inline README.md:225]: NOTE:  You're using denyNameStartsWith - it'd be good to call this out as a "excludedMetricsPrefix" OR perhaps change the metric filter to deny on that id ONLY.

Comment (shlomodaari) [reply README.md]: Changed the name :)

**Comment by christosarvanitis** [comment]
Thank @shlomodaari can you please add some tests on this change?

**Comment by shlomodaari** [comment]
Oh course :) Will work on it today! @christosarvanitis

**Comment by shlomodaari** [comment]
@christosarvanitis, are these tests okay with you? let me know if you need me to add another one :)

**Review by christosarvanitis — APPROVED** [review]

**Pull request merged**